omni-reader/README.md

```diff
@@ -0,0 +1,224 @@
+# OmniReader - Multi-model text extraction comparison
+
+OmniReader is a document processing workflow that ingests unstructured documents (PDFs, images, scans) and extracts text using multiple OCR models. It provides side-by-side comparison of extraction results, highlighting differences in accuracy, formatting, and content recognition. The multi-model approach allows users to evaluate OCR performance across different document types, languages, and formatting complexity. OmniReader delivers reproducible, automated, and cloud-agnostic analysis, with comprehensive metrics on extraction quality, processing time, and confidence scores for each model. It also supports parallel processing for faster batch operations and can compare an arbitrary number of models simultaneously.
+
+## 🚀 Getting Started
+
+### Prerequisites
+
+- Python 3.9+
+- Mistral API key (set as environment variable `MISTRAL_API_KEY`)
+- OpenAI API key (set as environment variable `OPENAI_API_KEY`)
+- ZenML >= 0.80.0
+
+### Installation
+
+```bash
+# Install dependencies
+pip install -r requirements.txt
+```
+
+### Configuration
+
+1. Ensure any Ollama models you want to use are pulled, e.g.:
+
+```bash
+ollama pull llama3.2-vision:11b
+ollama pull gemma3:12b
+```

```

Comment:
The ollama stuff comes maybe a bit out of left-field when I see it here? We probably need to mention that we are going to use ollama models for convenience or something?

In reality, in production use cases people would deploy these models in the non-GGUF versions since the performance / accuracy is so much better. So we probably need to address this by making it a 'runs on your local machine' kind of thing?